src/new-components/TopBar.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from '@emotion/styled'
+import { colorsV2 } from '@hedviginsurance/brand'
+import { HedvigLogo } from 'components/icons/HedvigLogo'
+import * as React from 'react'
+import { CurrentLanguage } from '../components/utils/CurrentLanguage'
+
+interface Props {
+  transparent?: boolean
+}
+
+const Wrapper = styled.div<{ transparent?: boolean }>`
```

Comment:
Lets extract this type 🙂

Reply:
Yeah sure, I used the same `Props` interface instead 😸